myJS.js

```diff
@@ -2,33 +2,70 @@ document.addEventListener("DOMContentLoaded", function () {
     const apiKey = '00b5bc3025c06669c4aecf0a745e058b';
     const searchForm = document.getElementById("search-form");
     const searchInput = document.getElementById("search-input");
-    let searchResults = document.getElementById("search-results");
+    const searchResults = document.getElementById("search-results");
+    let currPage=1;
+    const perPage=10;
 
     searchForm.addEventListener("submit", function (e) {
         e.preventDefault();
         const searchTerm = searchInput.value.trim();
         if (searchTerm) {
+            currPage=1
             searchFlickr(apiKey, searchTerm);
         }
     });
 
     function searchFlickr(apiKey, searchTerm) {
         // Make a request to the Flickr API
-        fetch(`https://www.flickr.com/services/rest/?method=flickr.photos.search&api_key=${apiKey}&text=${searchTerm}&per_page=10&sort=relevance&format=json&nojsoncallback=1`)
-            .then(response => response.json())
+        const page=currPage;
+        fetch(`https://www.flickr.com/services/rest/?method=flickr.photos.search&api_key=${apiKey}&text=${searchTerm}&per_page=${perPage}&page=${page}&sort=relevance&format=json&nojsoncallback=1`)
+            .then(response => response.json() )
             .then(data => {
+                console.log(data);
                 const photos = data.photos.photo;
                 searchResults.innerHTML = ""; // Clear previous results
                 photos.forEach(photo => {
                     const imageUrl = `https://farm${photo.farm}.staticflickr.com/${photo.server}/${photo.id}_${photo.secret}.jpg`;
                     const img = document.createElement("img");
                     img.src = imageUrl;
                     searchResults.appendChild(img);
+                    img.classList.add("photo");
+                    img.classList.add("search-result")
+                    addPaginationControls(data.photos.pages);
 
                 });
 
             })
             .catch(error => console.error("Error fetching data from Flickr:", error));
     }
 
+    function addPaginationControls(totalPages) {
+        // Create pagination controls (Next and Previous buttons)
+        const paginationContainer = document.getElementById("pagination");
+        paginationContainer.innerHTML = "";
+
+        if (currPage > 1) {
+            const prevButton = document.createElement("button");
```

Comment:
why adding the items using js? I think you can add them on your html and show/hide them using css.
it's more performant.
also you're adding/removing the elements and adding new event listeners without removing them. (removeEventListener)